addons/web/static/src/js/views/list/list_renderer.js

```diff
@@ -356,17 +356,18 @@ var ListRenderer = BasicRenderer.extend({
             var $el = this._renderFieldWidget(node, record, _.pick(options, 'mode'));
             return $td.append($el);
         }
+        this._handleAttributes($td, node);
         var name = node.attrs.name;
         var field = this.state.fields[name];
         var value = record.data[name];
-        var formattedValue = field_utils.format[field.type](value, field, {
+        var formatter = field_utils.format[field.type];
+        var formatOptions = {
             data: record.data,
-            escape: true,
             isPassword: 'password' in node.attrs,
-            digits: node.attrs.digits ? JSON.parse(node.attrs.digits) : undefined,
-        });
-        this._handleAttributes($td, node);
-        var title = field.type !== 'boolean' ? formattedValue : '';
+            digits: node.attrs.digits && JSON.parse(node.attrs.digits),
+        };
+        var formattedValue = formatter(value, field, _.defaults({escape: true}, formatOptions));
```

Comment:
why do you default escape true in formatoptions, but you know exactly what formatoptions is?